src/interface.rs

```diff
@@ -360,4 +361,38 @@ impl HWIClient {
             status.into()
         })
     }
+
+    /// Get installed version of hwilib. Returns None if not found, or if version couldn't be known
+    pub fn get_version() -> Option<String> {
+        match || -> Result<String, Error> {
+            Python::with_gil(|py| {
+                Ok(PyModule::import(py, "hwilib")?
+                    .getattr("__version__")?
+                    .to_string())
+            })
+        }() {
+            Ok(s) => Some(s),
+            Err(_) => None,
+        }
```

Comment:
Can be changed to 
```
            Some(
                PyModule::import(py, "hwilib")
                    .ok()?
                    .getattr("__version__")
                    .expect("Should have a __version__")
                    .to_string(),
            )
```

Which is more readable 

I think it's fine to `expect` on `__version__`, what do you think?